quarry/data/packets/0736_1.16.1.csv

```diff
@@ -0,0 +1,153 @@
+protocol_mode,packet_direction,packet_name
+init,upstream,handshake
+play,downstream,spawn_object
+play,downstream,spawn_experience_orb
+play,downstream,spawn_global_entity
```

Comment:
This packet was removed in 1.16, which is causing all packets afterwards to be incorrectly named